QualityControl/lib/database/repositories/LayoutRepository.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+/**
+ * LayoutRepository class to handle CRUD operations for Layouts.
+ */
+export default class LayoutRepository {
+  findLayoutById(data, layoutId) {
+    return data.layouts.find((layout) => layout.id === layoutId);
```

Comment:
My personal preference is to use `null` when meaning `no data`, not undefined (as it is the case here). 
That is my personal opinion, the best is to stick to the project's conventions.